src/screens/auth/login/Login.tsx

```diff
@@ -155,6 +160,26 @@ export default class Login extends BaseScreen<Props, State> {
     }
   }
 
+  private noTenantFoundMessage(){
+    Alert.alert(
+      I18n.t('authentication.noTenantFoundTitle'),
+      I18n.t('authentication.noTenantFoundMessage'),
+      [
+        {text: I18n.t('authentication.addTenantButton') , onPress:() => {this.goToTenants(true)}},
+        {text: I18n.t('general.close'), style: 'cancel' }
+      ]
+    )
+  }
+
+  private goToTenants(openQRCode = false) {
+    this.props.navigation.navigate(
+      'Tenants',
+      {
+        key: `${Utils.randomNumber()}`,
+        openQRCode
```

Comment:
No need to pass a boolean I think, You can jump right away to the QR-Code screen via the Tenant one.
Check as an example of navigation screen in screen in NotificationManager:
```
      this.navigator.dispatch(
          StackActions.replace(
            'AppDrawerNavigator',
            {
              screen: 'TransactionHistoryNavigator',
              initial: false,
              params: {
                screen: 'TransactionDetailsTabs',
                key: `${Utils.randomNumber()}`,
                params: {
                  params: {
                    transactionID: Utils.convertToInt(notification.data.transactionId)
                  }
                }
              }
            }
          ),
        );

```

Reply:
I don't think it is possible for this, the opening is already handled with a boolean inside the Tenant screen

Reply:
Yes you're right, this is part of the Tenant's list.